src/layouts/InterfaceLayout/InterfaceLayout.vue

```diff
@@ -495,9 +495,15 @@ export default {
     },
     setENS() {
       if (this.wallet.identifier === 'Web3') {
```

Comment:
This is if-else is redundant, must be a left over from a previous code
please remove the if-else and set the ens inside